layouts/_default/single.html

```diff
@@ -51,7 +51,7 @@
 					{{ if and (eq .Language.Lang "zh-cn") ( not .Page.Params.untranslated ) }}
 					<p data-langswitch="{{ .Language.Lang }}" style="display:none">
 						<a lang="zh-cn" href="{{ .Permalink }}">
-							使用简体中文阅读本网页。
+							阅读中文页面
```

Comment:
[zh-tw] have been merged so now in the file you have both: 
```
						<a lang="zh-cn" href="{{ .Permalink }}">
							使用简体中文阅读本网页。
						</a>
```
and
```
						<a lang="zh-tw" href="{{ .Permalink }}">
							使用正體中文閲讀本網頁。
						</a>
```
So I think your suggestion is not a good idea. We must keep the differentiation.